### ISSUE TITLE
Multiple tool tip is visible for the help icon button present under Edit button tile.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#118]
User Experience:

If multiple tooltip is defined for and icon button, then the user will get confused and unable to understand the purpose of the control.

Note: User credentials should NOT be included in the bug.

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Button and select it.
8. Navigate to the "Button" dialog and select edit button.
9. Navigate to the various help icon controls of the "Button" dialog box like label, link to, appended, size etc.
10. Verify the issue.

 

Actual Result:

Multiple tool tip is visible for the help icon button present under Edit button tile.

 

Expected Result:

Only one tool tip is visible for each control present under Edit button tile so that user can get the correct information.

Note:

Similar issue exists with Chrome.

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
